### ISSUE TITLE
Replace registries administered community groups with registries established by this specification

### ISSUE THREAD

**Issue by selfissued** [w3c/did-core#169]
Section 6.3 on Public Keys https://w3c.github.io/did-core/#public-keys contains this normative requirement requiring use of a registry operated by a community group:
> All public key properties MUST be from the Linked Data Cryptographic Suite Registry.

We should replace uses of community group registries with registries established by this specification and governed by this working group.  It's in appropriate for a specification destined for Recommendation status to utilize registries defined and operated by community groups, as their long-term stability is not assured.  This should definitely happen before we reach Proposed Recommendation status.

**Comment by selfissued** [comment]
There appeared to be agreement to do this on the latest registries call.

**Comment by OR13** [comment]
We are in progress of doing this, in the https://github.com/w3c/did-core-registry

**Comment by jandrieu** [comment]
Actually, I think this is a misplaced interpretation of W3C policy. 

The requirement is that normative registries are set up by working groups, but they have consistently endorsed community groups as long-term vehicles for maintaining those registries. 

There are proposals for "evergreen" working groups, but the update to W3C registry process has just recently been pushed to the next update to W3C process, not this process: per https://www.w3.org/2020/01/29-w3process-minutes.html and https://www.w3.org/2020/02/12-w3process-minutes.html

*This* working group is not evergreen and it *will* need to assign governance authority to another entity. The consensus to date has been to use the CCG for that, as it, in fact, is capable of surviving indefinitely. It is also what was done with the registries required for Verifiable Credentials.

**Comment by iherman** [comment]
> There are proposals for "evergreen" working groups, but the update to W3C registry process has just recently been pushed to the next update to W3C process, not this process: per https://www.w3.org/2020/01/29-w3process-minutes.html and https://www.w3.org/2020/02/12-w3process-minutes.html

that is correct. However, it is perfectly possible that, by the time this WG finishes, we set up a continuation WG for maintenance, just as we have done for VC. That WG _may_ (this is still open) do a continuous maintenance, like "evergreen", for the DID specification, and may also serve as a long-term maintenance/governance group for the registry.

I am not saying this _should_ the way. I am just saying that the decision on whether it is the CCG or the maintenance version of this WG should/could be decided later. This WG should publish the registry document itself, outlining the process, etc; we can obviously publish this document as a Note but it is not impossible to publish it as a Rec if we so desire.

My feeling is that we should postpone this decision. The Proposed Rec status of this specification is a year away after all...

**Comment by rhiaro** [comment]
Is the next step a PR to replace the text about the LD Cryptosuites Registry with something like:

> Available public key types are listed in the [[DID-CORE-REGISTRIES]]. Public key types not listed in the [[DID-CORE-REGISTRIES]] MUST NOT be used.

**Comment by selfissued** [comment]
The specification still contains references to https://w3c-ccg.github.io/did-method-registry/ and https://w3c-ccg.github.io/ld-cryptosuite-registry/.  These either need to be replaced by registries administered by this working group, another bona fide standards organization, such as the IETF, or deleted.

**Comment by OR13** [comment]
did method registry is getting merged into did core registries last comment here indicates the CCG is waiting on Chairs of the DID WG: https://github.com/w3c-ccg/did-method-registry/issues/82

DID Core Registries will retain references to:
- https://w3c-ccg.github.io/ld-cryptosuite-registry/
- https://w3c-ccg.github.io/security-vocab/

There 2 lines need to point at DID Core Registries instead:

https://github.com/w3c/did-core/blob/master/index.html#L3231

https://github.com/w3c/did-core/blob/master/index.html#L3242
